tests/models/modules/test_masked_autoencoder.py

```diff
@@ -84,7 +88,9 @@ def test_forward_cuda(self):
         self._test_forward(torch.device("cuda"))
 
 
-@unittest.skipUnless(_torchvision_vit_available, "Torchvision ViT not available")
+@unittest.skipUnless(
+    dependency.torchvision_vit_available(), "Torchvision ViT not available"
+)
 class TestMAEDecoder(unittest.TestCase):
     def test_init(self):
         return MAEDecoder(
```

Comment:
FWIW, we should prob remove this return statement:
```
.../lib/python3.11/unittest/case.py:678: DeprecationWarning: It is deprecated to return a value that is not None from a test case (<bound method TestMAEDecoder.test_init of <test_masked_autoencoder.TestMAEDecoder testMethod=test_init>>)
    return self.run(*args, **kwds)
```

Reply:
Thanks, fixed! We anyways want to move test to pytest since a while :)